tests/commands/new/test_make_class_name.py

```diff
@@ -9,7 +9,7 @@
         ('Hello! World', 'HelloWorld'),
         ('Hello_World', 'Hello_World'),
         ('Hello-World', 'HelloWorld'),
-        ('24 Jump Street', '_24JumpStreet'),
```

Comment:
Since this whole PR exists because of a specific bug report, we should include the case from that bug report in the testing.

Reply:
Good point. I added a couple of tests using non-latin characters. My changes have passed these tests, but let me know if you had anything more specific in mind and I will be glad to make the changes.